index.js

```diff
@@ -16,6 +16,12 @@ const users = [
       "⚛️ Senior JS Developer @TelusDigital || @FreeCodeCamp core || Working on my own garden. Cannot care if your side of the hedge seems greener. || INTJ-A Úlfhéðin",
     github: "@hallaathrad"
   }
```

Comment:
This array doesn't work. Did you test it locally? It seems it's missing something.